.github/workflows/kind.yml

```diff
@@ -189,6 +189,60 @@ jobs:
         path: log.tar.gz
         retention-days: 30
 
+  test-e2e-encap-endpointslice:
```

Comment:
@weiqiangt @hongliangl Just want to confirm with you that this is a useful addition to the test list. I believe that because Kind v0.9.0 uses K8s 1.19, EndpointSlice Mirroring is enabled, so we should be exercising at least some of the new code for those e2e tests for which we use Services?

(let's make sure we run the `jenkins-all-features-conformance` on this PR, as that should provide some coverage)

Reply:
The endpointslice support is controlled by feature-gate, I think we need to explicitly enable it to cover e2e tests.

The all-features test has been passed https://jenkins.antrea-ci.rocks/job/antrea-all-features-conformance-for-pull-request/274/ .

Reply:
> The endpointslice support is controlled by feature-gate, I think we need to explicitly enable it to cover e2e tests.

Just to clarify, for the new `test-e2e-encap-endpointslice` Kind test:
- I believe we are using K8s 1.19, which should have EndpointSlice enabled by default
- We are generating a YAML manifest which enables EndpointSlice support in AntreaProxy
so to some extent, that Kind test should be exercising EndpointSlice. Is that not the case?

Reply:
Sorry, I'm not sure I actually get the point.
You mean we need add specific EndpointSlice tests? But once we created a Service with pod selector in kubernetes cluster with EndpointSlice enabled, either EndpointSlices or Endpoints can be used to read the Endpoint data. If we enabled the feature gate, then we watch EndpointSlice, otherwise Endpoints.

Reply:
I'm trying to make sure that this new Kind test is useful, i.e. trying to make sure that it does exercise some of the EndpointSlice support in AntreaProxy. If it does, then great. If it doesn't, then we don't need to add it. My guess is that it is useful based on the reasons I listed above, but I wanted to double check with you.

Reply:
Got it. This test does cover this commit since we only watch EndpointSlice when the feature gate is enabled, otherwise tests about Services in e2e will fail.

Reply:
@tnqn This feature is transparent to Service test in `jenkins-all-features-conformance`. But more test characteristics can be defined on e2e for now or in the future. For now, the test can be deleted in on e2e.